### ISSUE TITLE
HubSpot not mapping name field

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#535]
**Description**

I'm seeing forms failing to sync to HubSpot due to the required name field being reported as empty.

Here is the error from the queue:
```
API error: “{"status":"error","message":"The request is not valid","correlationId":"1ae56c6e-846e-40b9-8d10-21e72c209a7f","errors":[{"message":"Error in 'fields.firstname'. Required field 'firstname' is missing","errorType":"REQUIRED_FIELD"},{"message":"Error in 'fields.lastname'. Required field 'lastname' is missing","errorType":"REQUIRED_FIELD"}]}” /var/app/current/cms/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:113
```

The first name and last name subfields are filled out in the submission and mapped to a form submission in HubSpot (not a contact).

The job also reports the payload as:
```
    "payload": {
        "fields": [
            {
                "name": "email",
                "value": "<redacted>"
            }
        ]
    },
```

so it doesn't appear the name is even included in the payload.

 I've tried refreshing the integration, resaving it, etc but haven't had any luck.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes 
- Custom Form Templates: Partial

**Additional info**
- Plugin version: 1.4.12
- Craft version: 3.7.7 Pro
- Multi-site: No

**Additional context**
PHP 8 on Amazon Linux 2

**Comment by engram-design** [comment]
If you can confirm you name field settings (handle, multi or not, enabled sub-fields), and your mapping settings, I can double check.

Also - what integration setting you're using (mapping to a contact, form, etc)

**Comment by engram-design** [comment]
Actually, figured it out. Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.13](https://github.com/verbb/formie/releases/tag/1.4.13)
